docs/registries.md

```diff
@@ -16,16 +16,22 @@ To support private GitHub, simply authenticate with your private GitHub account:
 ```
 
 ```
-Would you like to set up your GitHub credentials? [yes]: 
+Would you like to set up your GitHub credentials? [yes]:
      If using two-factor authentication or to avoid using your password you can generate an access token at https://github.com/settings/applications.
 
 Enter your GitHub username: username
-Enter your GitHub password or access token: 
-Would you like to test these credentials? [yes]: 
+Enter your GitHub password or access token:
+Would you like to test these credentials? [yes]:
 ```
 
 This will enable private repo installs.
 
+#### Github Rate Limiting
```

Comment:
I had used `Continuous Integration` first, but this also applies to services like Heroku as well as people just randomly behind proxies and firewalls.